### PR TITLE
add teacher announcement for updates to hints system

### PR DESCRIPTION
**Pull request by tanyaparker**
**teacher dashboard**
image

**Code Studio homepage**
image

### CODE REVIEW

File: dashboard/app/views/api/teacher_announcement.html.haml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  %a.announce-heading-link{:href=>"http://teacherblog.code.org/"}
+    %div.announce-heading
+      = t('home.announcement_title')
+      %div.heading-view-all
+        = t('home.announcement_view_all')
+        >>
+  %div.announce-content
+    %a{:href=>cta_url}
+      %div.content-img-container
+        = image_tag(CDO.shared_image_url("/teacher-announcement/tagging-puzzles.jpg"), width: '200px')
```
Comment (breville) [inline dashboard/app/views/api/teacher_announcement.html.haml:15]: this image is quite large.. can we scale it down first?

File: pegasus/sites.v3/code.org/views/teacher_announcements.haml [hunk 2]
```diff
@@ -9,6 +10,15 @@
         = I18n.t(:dashboard_announce_view_all)
         >>
   %div.announce-content
+    %a{:href=>cta_url}
+      %div.content-img-container
+        %img.content-img{:src=>"/images/fill-300x200/teacher-dashboard/tagging-puzzles.jpg"}
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/teacher_announcements.haml:15]: good that we're scaling the huge image here.

**Comment by breville** [comment]
one comment, otherwise lgtm

File: dashboard/app/views/api/teacher_announcement.html.haml (hunk 1 above)
Comment (tanyaparker) [inline dashboard/app/views/api/teacher_announcement.html.haml:15]: Oh yeah, of course!

**Pull request merged**